expression/explain.go

```diff
@@ -170,6 +209,27 @@ func SortedExplainNormalizedExpressionList(exprs []Expression) []byte {
 	return sortedExplainExpressionList(exprs, true)
 }
 
+// SortedExplainNormalizedExpressionList4InList generates explain information for a list of expressions in order.
+func SortedExplainNormalizedExpressionList4InList(exprs []Expression) []byte {
+	buffer := bytes.NewBufferString("")
+	exprInfos := make([]string, 0, len(exprs))
+	for _, expr := range exprs {
+		if _, ok := expr.(*ScalarFunction); ok {
+			exprInfos = append(exprInfos, expr.ExplainNormalizedInfo4InList())
+		} else {
+			exprInfos = append(exprInfos, expr.ExplainNormalizedInfo())
```

Comment:
Looks like this branch is unnecessary. And this entire function can be merged into `sortedExplainExpressionList()`.